lamindb/_artifact.py

```diff
@@ -296,7 +305,7 @@ def get_relative_path_to_directory(
 
 def get_artifact_kwargs_from_data(
     *,
-    data: Path | UPath | str | pd.DataFrame | AnnData,
+    data: Path | UPath | str | pd.DataFrame | AnnData | MuData,
```

Comment:
Also this might be problematic.

Reply:
All CI doesn't have mudata installed except for docs

Reply:
OK, great!